risc0/zkp/src/hal/metal.rs

```diff
@@ -850,49 +850,49 @@ impl<MH: MetalHash> Hal for MetalHal<MH> {
         self.hash.as_ref().unwrap().get_hash_suite()
     }
 
-    #[cfg(feature = "metal_prefix_products")]
-    fn prefix_products(&self, io: &Self::Buffer<Self::ExtElem>) {
-        let block_size = 256;
-        let grain_size = 16;
```

Comment:
Wondering why these are changing for a backport?

Reply:
I backported the toolchain changes to get the CI to work and newer toolchains detect features that aren't in the Cargo.toml like this `metal_prefix_products` feature